src/hooks/useContract.ts

```diff
@@ -30,7 +30,7 @@ import WETH_ABI from 'constants/abis/weth.json'
 import ZAP_STATIC_FEE_ABI from 'constants/abis/zap-static-fee.json'
 import ZAP_ABI from 'constants/abis/zap.json'
 import { MULTICALL_ABI } from 'constants/multicall'
-import { EVM_NETWORK, isEVM } from 'constants/networks'
+import { EVM_NETWORK, NETWORKS_INFO_CONFIG, isEVM } from 'constants/networks'
```

Comment:
Dont use `NETWORKS_INFO_CONFIG` directly. It not intend to be use directly. Use `NETWORKS_INFO` instead